docs/aws-professional/services.md

```diff
@@ -22,168 +22,168 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS exper
-| Scalability                     | AWS Auto Scaling                | [Virtual Machine Scale Sets](/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-overview) <br/><br/>[Azure App Service Scale Capability (PAAS)](https://azure.microsoft.com/documentation/articles/web-sites-scale/) <br/><br/>[Azure AutoScaling](/azure/app-service/app-service-environment-auto-scale) | Lets you automatically change the number of instances providing a particular compute workload. You set defined metric and thresholds that determine if the platform adds or removes instances.                                                  |
-| Predefined templates            | AWS Quick Start                 | [Azure Quickstart templates](https://azure.microsoft.com/documentation/templates/)                                                                                                                                                                                                                                                                                           | Community-led templates for creating and deploying virtual machine–based solutions.                                                                                                                                                             |
+| Virtual servers                 | Elastic Compute Cloud (EC2) VMs | [Virtual Machines](https://azure.microsoft.com/services/virtual-machines/)                                                                                                                                                                                                                                                                                       | Allows users to deploy, manage, and maintain OS and server software; instance types provide configurations of CPU/RAM.                                           |
+| **&nbsp;**                      | Amazon Lightsail                | [Virtual Machines & Images](https://azure.microsoft.com/services/virtual-machines/)                                                                                                                                                                                                                                                                              | Offers a lightweight, simplified product offering users can choose from from when building out a virtual machine.                                                                                                                                               |
```

Comment:
Not sure what "lightweight" means in the context

Reply:
lightweight is referred to in this context as "you pick the software" that you want installed. It also has been noted that "community-led templates" isn't accurate for Lightsail.